packages/ng/callout/callout-feedback-list/callout-feedback-list.component.scss

```diff
@@ -0,0 +1 @@
+@use '@lucca-front/scss/src/components/calloutFeedbackList';
```

Comment:
We use the same stylesheet twice? The style will be duplicated in the DOM. Could we split `@use '@lucca-front/scss/src/components/calloutFeedbackList'` In two files?

Reply:
ping @jeremie-lucca (it's used in both list and item components)